torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1754,22 +1754,90 @@ std::tuple<at::Tensor, at::Tensor> FoldConvBatchNorm2dHelper::
   return std::make_tuple(new_w, new_b);
 }
 
+std::vector<c10::optional<IValue>> findEPSWeightBiasConstantsOfBN(
+    const script::Module& bn) {
+  auto graph = bn.get_method("forward").graph();
+  const PatternInfo& pattern_bn = PatternInfo::parse_from_str(R"(
+      graph(%a, %weight, %bias, %running_mean, %running_var,
+          %training, %momentum, %eps, %cudnn_enabled):
+        %bn_out = aten::batch_norm(%a, %weight, %bias, %running_mean,
+            %running_var, %training, %momentum, %eps, %cudnn_enabled)
+        return (%bn_out) )");
+  const Graph& pattern_bn_graph = *pattern_bn.pattern_graph;
+  const auto& bn_vmap = pattern_bn.vmap;
+
+  const auto& matches = findPatternMatches(pattern_bn_graph, *graph);
+  if (matches.size() == 0) {
+    return std::vector<c10::optional<IValue>>(3, c10::optional<IValue>{});
+  }
+  const auto& match = matches[0];
+  std::vector<c10::optional<IValue>> eps_weight_bias_constants;
+  eps_weight_bias_constants.reserve(3);
+  eps_weight_bias_constants.emplace_back(toIValue(match.values_map.at(bn_vmap.at("eps"))));
+  eps_weight_bias_constants.emplace_back(toIValue(match.values_map.at(bn_vmap.at("weight"))));
+  eps_weight_bias_constants.emplace_back(toIValue(match.values_map.at(bn_vmap.at("bias"))));
+  return eps_weight_bias_constants;
+}
+
+bool extractOptionalBNParams(
+    const script::Module& bn,
+    ConvBNParameters& r) {
+  auto eps_weight_bias_constants = findEPSWeightBiasConstantsOfBN(bn);
+
+  if (bn.hasattr("eps")) {
+    r.bn_eps = bn.attr("eps").toDouble();
+  } else {
+    auto optional_eps = eps_weight_bias_constants[0];
```

Comment:
Nit: it would be easier to read if `findEPSWeightBiasConstantsOfBN` returned a map:
```
auto optional_eps = eps_weight_bias_constants.at("eps");
```